lib/gtfs/data.ex

```diff
@@ -205,8 +205,13 @@ defmodule Gtfs.Data do
 
   @spec parse_files(all_files()) :: t()
   def parse_files(%{gtfs: gtfs_files, hastus: hastus_files}) do
+    _hastus_activities = Gtfs.Hastus.Activity.parse(hastus_files["activities.csv"])
+    hastus_trips = Gtfs.Hastus.Trip.parse(hastus_files["trips.csv"])
+
     directions_by_route_id = directions_by_route_id(gtfs_files["directions.txt"])
-    run_ids_by_trip_id = Run.run_ids_by_trip_id(hastus_files["trips.csv"])
+
+    run_ids_by_trip_id =
+      Map.new(hastus_trips, fn hastus_trip -> {hastus_trip.trip_id, hastus_trip.run_id} end)
```

Comment:
What if `Gtfs.Hastus.Trip.parse` just returned this format?

Reply:
When grouping things into minischedules, we'll want to work with the ordered list of hastus trips, so returning the map would get in the way of that.